src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/FunctionsTests.cs

```diff
@@ -2,9 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Collections.Generic;
+using System.IO;
```

Comment:
Are these `using`s still needed ?

Reply:
The new test uses Path.Combine to build a path, I know in other places the code just manually creates a path but I think Path.Combine is better (I was going to do another pr converting the manually built strings to Path.Combine)